ychat/src/commonMain/kotlin/co/yml/ychat/data/api/impl/ChatGptApiImpl.kt

```diff
@@ -62,4 +65,14 @@ internal class ChatGptApiImpl(private val apiExecutor: ApiExecutor) : ChatGptApi
             .setHttpMethod(HttpMethod.Get)
             .execute()
     }
+
+    override suspend fun audioTranscriptions(audioParamsDto: AudioParamsDto): ApiResult<AudioResultDto> {
+        val byteArray = audioParamsDto.byteArray.toByteArray()
+        val apiBuilder = apiExecutor
+            .setEndpoint("v1/audio/transcriptions")
```

Comment:
I think we could extract the `/v1` in an upcoming PR to a `VERSION` constant, so we can eventually refactor easily.

Reply:
I made the adjustments in this PR. Extracted to a `VERSION` constant